mariadb/migrate.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Needs to be verified that the ranges are correct...
+source=$(echo $VCAP_SERVICES | jq -r '."mariadb-k8s-database"[] | select(.credentials.host | contains("144"))')
+dest=$(echo $VCAP_SERVICES | jq -r '."mariadb-k8s-database"[] | select(.credentials.host | contains("197"))')
+
+# Source
+$SRC_HOST=$(echo $source | jq -r '.credentials.host')
+$SRC_DATABASE=$(echo $source | jq -r '.credentials.database')
+$SRC_USER=$(echo $source | jq -r '.credentials.username')
+$SRC_PASSWORD=$(echo $source | jq -r '.credentials.password')
+
+# Destination
+$DST_HOST=$(echo $dest | jq -r '.credentials.host')
```

Comment:
Would be good to check that these values are not empty before passing them to the mariadb-dump and mariadb commands